cdap-docs/examples-manual/build.sh

```diff
@@ -154,9 +154,9 @@ function download_includes() {
   test_an_include 2176f426a50b2d68817cd7da7b7faffb ../../cdap-examples/StreamConversion/src/main/java/co/cask/cdap/examples/streamconversion/StreamConversionApp.java
   test_an_include b3ad7dc7ddbb5b905145dbf15a5c9b93 ../../cdap-examples/StreamConversion/src/main/java/co/cask/cdap/examples/streamconversion/StreamConversionMapReduce.java
   
-  test_an_include dc6519d102875a83621f262c33c61894 ../../cdap-examples/UserProfiles/src/main/java/co/cask/cdap/examples/profiles/UserProfiles.java
+  test_an_include 2260781c7cef2938aa36c946f3a34341 ../../cdap-examples/UserProfiles/src/main/java/co/cask/cdap/examples/profiles/UserProfiles.java
```

Comment:
@anew 's explanation is correct. Sometimes the file has changed but it doesn't affect the include, and so I just update the MD5 hash. With this change, the files actually did change such that the include needed to be altered. It is manual work, but I haven't figured out a better solution.